01_Introduction/intro.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>Don't use abstract i notation in explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>I forgot what this means</p><p> </p>